backend/danswer/connectors/notion/connector.py

```diff
@@ -0,0 +1,222 @@
+"""Notion reader."""
+import time
+from dataclasses import dataclass, fields
+from typing import Any, Dict, List, Optional
```

Comment:
If you get a chance, can you run the format/linter steps?

reorders the imports to be 1 per line, also there are some mypy issues, and I think 1 line of change from black formatter.
Also if you can run the frontend prettify.

If all this is a hassle to set up, just let me know, we can put it in and I'll run the formatter etc.